Ref/test/int/ref_integration_test.py

```diff
@@ -134,10 +143,38 @@ def test_send_command(self):
         self.assert_command("cmdDisp.CMD_NO_OP", max_delay=0.1)
         assert self.api.get_command_test_history().size() == 2
 
+    def test_send_command_args(self):
+        for count, value in enumerate(["Test String 1", "Some other string"], 1):
+            events = self.api.get_event_pred("cmdDisp.NoOpStringReceived", [value])
```

Comment:
Are you intending to assert that this event is emitted? To my understanding this event object look unused.